packages/frontend/translations/fr-FR.yaml

```diff
@@ -103,6 +103,9 @@ models:
   identity:
     name: Nom
     publicKey: Clé publique
+    verification: Statut de vérification
```

Comment:
you don't need to worry about these other languages, crowdin's automatic github integration will PR for these